CMakeLists.txt

```diff
@@ -89,7 +89,13 @@ set(LLVM_EXTERNAL_LIT ${LLVM_TOOLS_BINARY_DIR}/llvm-lit CACHE STRING "Command us
 
 if(LINK_WITH_FIR)
   include(TableGen)
+
+  find_package(MLIR REQUIRED CONFIG)
+  # Use SYSTEM for the same reasons as for LLVM includes
+  include_directories(SYSTEM ${MLIR_INCLUDE_DIRS})
+  list(APPEND CMAKE_MODULE_PATH ${MLIR_DIR})
```

Comment:
Is it important that these are done before `include(AddMLIR)`?

Reply:
I just followed the order from llvm/llvm-project@7ca473a commit message. I did not try moving them afterwards, would it be better ?

Reply:
No. I was just wondering if some settings like this are causing the test failures.